### ISSUE TITLE
Add App Privacy Manifest?

### ISSUE THREAD

**Issue by clausjoergensen** [facebook/yoga#1551]
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files

**Comment by NickGerleman** [comment]
Makes sense. Not sure what Coocoapods/SwitfPM integration would look like but seems easy enough. Yoga does not collect any sort of user data.
